rails/db/migrate/20200726232543_create_themes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateThemes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :themes do |t|
+      t.string :background_img
+      t.boolean :active
```

Comment:
Lets change this to set a database constraint to `null: false` - this way when we are utilizing the boolean in Rails, it'll always be a true boolean (instead of a 3-way "boolean" - true / false / null [falsey]).

```suggestion
      t.boolean :active, null: false, default: false
```

Here is more thoughts on the threestate boolean problem: https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem.